roles/openshift_node/tasks/main.yml

```diff
@@ -22,7 +22,7 @@
       deployment_type: "{{ openshift_deployment_type }}"
   - role: node
     local_facts:
-      labels: "{{ openshift_node_labels | default(none) }}"
+      labels: "{{ lookup('oo_option', 'openshift_node_labels') | default( openshift_node_labels | default() ) }}"
```

Comment:
`labels: "{{ lookup('oo_option', 'openshift_node_labels') | default( openshift_node_labels | default(none) , true) }}"`
With @twiest 's adivce, what do you think about this ?